### ISSUE TITLE
Update Bulma to latest dist

### ISSUE THREAD

**Issue by KonradDeskiewicz** [vue-bulma/vue-admin#215]
Hi,

There are great goodies in Bulma 0.4.x .
Any chance to update your admin?
I'm targeting Bulma 0.4.4 especially because of button drop-downs that are "Swiss knife" in admin panels.

**Comment by acampagnaro** [comment]
What it is that you want, but you are trying this very difficult.

**Comment by KonradDeskiewicz** [comment]
I suppose there are issues with update Bulma's package, but sticking to Bulma 0.3.x cause this admin to be more and more obsolete.

**Comment by KonradDeskiewicz** [comment]
It was just bumped to 0.5 / 0.5.1 with lot of changes - classes especially.

**Comment by Ortix92** [comment]
+1

**Comment by DevparkBartekg** [comment]
@KonradDeskiewicz Hi
Did you update bulma to 0.4.x?

**Comment by KonradDeskiewicz** [comment]
No. We've dropped VueAdmin as this project is not reliable anymore. We're using another admin based on BootstrapVue that is far much better maintained.

**Comment by 0pt1m1z3r** [comment]
@KonradDeskiewicz what are you use?

**Comment by KonradDeskiewicz** [comment]
If you're looking for active repo and Bulma is not the only choice i strongly suggest [CoreUI - vue edition](https://github.com/mrholek/CoreUI-Vue) based on [bootstrap-vue](https://github.com/bootstrap-vue/bootstrap-vue). 
As bootstrap 4 is stable and "flex-driven" this repo is an obvious admin panel choice for now.
